### PR TITLE
Hash table and Linear probe Resolution method for Customer Details

### PR DESCRIPTION
**Pull request by Arun9739**
## Related Issue
- #1374 

## Proposed Changes
- Added a file for Hash Table of Customer details
- Updated the Readme accordingly

## Additional Info
- The code was tested well and the results are in well manner.

To store a customer details and search them in well manner, we use a Hash Table. In Hash table we enter the data in Linear hashing method and search them based on the same method. if we encounter any collision, we resolve it by using a Linaer probe resolution method.

The customer details contain :
- Customer name
- Customer Id
- customer phone

The functions used in the program : 
- insert() : To insert a data into the HashTable
- search() : To search whether a data is present in the hash table
- display() : To dispaly the Hash Table 

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
| Screenshot #1      | 
| ----------- | 
| image     |

### CODE REVIEW

**Comment by Arun9739** [comment]
@Mrjoy832 Please review my pr

This is my third pr. My first two successful pr's are :
- #1046 
- #1152

**Comment by Mrjoy832** [comment]
Keep a short name od CPP file

**Comment by Arun9739** [comment]
@Mrjoy832 renamed the cpp file succesfully

**Comment by Mrjoy832** [comment]
> @Mrjoy832 renamed the cpp file succesfully

check imp-announcement in LearnCPP Discord channel

File: H-HashTable/README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+3) else,
+    print no hash entry
+4) end loop
+
+### Time complexity
+
+```
+- Searching : O(1)
+- Insering : O(1)
+```
```
Comment (Mrjoy832) [inline H-HashTable/README.md:69]: add advantages and disadvantages

**Review by Mrjoy832 — CHANGES_REQUESTED** [review]

**Comment by Arun9739** [comment]
@Mrjoy832 done the changes sir

**Review by Lakhankumawat — APPROVED** [review]
LGTM ✨🎉

**Comment by Lakhankumawat** [comment]
Good Contribution .

**Pull request merged**